gatsby/src/pages/404.tsx

```diff
@@ -1,12 +1,26 @@
 import React from "react";
 import { Layout } from "../components/Layout/Layout";
 import { SEO } from "../components/SEO/SEO";
+import { Link } from "gatsby";
 
 const NotFoundPage: React.FC = () => {
 	return (
 		<Layout>
 			<SEO title="Page not found" noIndex={true} />
```

Comment:
[After talking with Nick](https://nicedigital.atlassian.net/wiki/spaces/CKS/pages/1430388923/CKS+404+page?focusedCommentId=1465516083#comment-1465516083), we decided the page should have a breadcrumb of NICE -> CKS -> Page not found to make it clear you're within CKS. Please can you add the breadcrumbs and we'll udpate the card?